tiledb/sm/rest/rest_client.h

```diff
@@ -294,6 +322,16 @@ class RestClient {
     return rest_server_;
   }
 
+  /// Operation disabled in base class.
+  inline virtual const TileDBVersion& rest_version() {
+    throw RestClientDisabledException();
+  }
+
+  /// Operation disabled in base class.
+  inline virtual const TileDBVersion& rest_minimum_supported_version() {
+    throw RestClientDisabledException();
+  }
+
```

Comment:
I would make these versions `const`-qualified, and the `RestClientRemote::rest_capabilities_` field `mutable`.

Reply:
Done